classes/UpgradeTools/CoreUpgrader/CoreUpgrader17.php

```diff
@@ -28,6 +28,9 @@
 namespace PrestaShop\Module\AutoUpgrade\UpgradeTools\CoreUpgrader;
 
 use PrestaShop\Module\AutoUpgrade\UpgradeException;
+use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
```

Comment:
I don't know if we are allowed to use core classes like this in the upgrade module Especially since these classes may not be available in the source version and/or the target version.

Reply:
Afaik you can do it:
![image](https://user-images.githubusercontent.com/1462701/140719612-f6689a08-1ddd-4a19-94fa-2bc797274748.png)
It will only raise an error if the class is not found and executed by PHP